static/panes/compiler.ts

```diff
@@ -1487,7 +1493,10 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.updateDecorations();
 
         const codeLenses: monaco.languages.CodeLens[] = [];
-        if (this.getEffectiveFilters().binary || result.forceBinaryView) {
+        if (this.getEffectiveFilters().binary
+            || this.getEffectiveFilters().binaryobject
+            || this.getEffectiveFilters().binaryobject
```

Comment:
you check `binaryobject` twice here redundantly.

Reply:
Woops, rebase gone wrong I guess